src/legacy/core_plugins/vis_type_vislib/public/vislib/helpers/point_series/_get_series.ts

```diff
@@ -18,28 +18,28 @@
  */
 
 import _ from 'lodash';
-import { getPoint } from './_get_point';
-import { addToSiri } from './_add_to_siri';
+import { getPoint, Point } from './_get_point';
+import { addToSiri, Serie } from './_add_to_siri';
+import { Chart } from './point_series';
+import { Table } from '../../types';
 
-export function getSeries(table, chart) {
+export function getSeries(table: Table, chart: Chart) {
   const aspects = chart.aspects;
   const xAspect = aspects.x[0];
   const yAspect = aspects.y[0];
-  const zAspect = aspects.z && aspects.z.length ? aspects.z[0] : aspects.z;
+  const zAspect = aspects.z && aspects.z[0];
   const multiY = Array.isArray(aspects.y) && aspects.y.length > 1;
-  const yScale = chart.yScale;
```

Comment:
Since `chart` object is created in `point_series` files and there is no initializing of `yScale`, I remove all code related to `yScale`